src/test/java/core/basesyntax/dao/RecordDaoTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.database.Database;
+import core.basesyntax.model.Record;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class RecordDaoTest {
+    private static final int ADD_CORRECT_DB_SIZE = 1;
+    private static final int GET_CORRECT_DB_SIZE = 3;
+    private static RecordDao recordDao;
+
+    @BeforeClass
+    public static void beforeAll() {
+        recordDao = new RecordDaoImpl();
+    }
+
+    @Test
+    public void addRecord_correctData_Ok() {
+        Record testRecord = new Record("b", "apple", 100);
+        recordDao.addRecord(testRecord);
+        int expected = ADD_CORRECT_DB_SIZE;
+        int actual = Database.RECORDS.size();
+        Assert.assertEquals("Database size should be"
+                + ADD_CORRECT_DB_SIZE, expected, actual);
+    }
+
+    @Test
+    public void getRecords_Ok() {
```

Comment:
Use single style for test cases' naming. Compare naming in this test case and the test case above.

Reply:
fixed